### PR TITLE
fix: some args are not used in cli.py

### PR DESCRIPTION
**Pull request by eggry**
`--num-gpus` is not used. Change to `--device` style to consist with `model_worker.py`.
`--temperature` and `--max-new-tokens` are not used when inference.

### CODE REVIEW

File: llava/serve/cli.py [hunk 1]
```diff
@@ -52,7 +52,6 @@ def main(args):
         roles = conv.roles
 
     image = load_image(args.image_file)
-    image_tensor = image_processor.preprocess(image, return_tensors='pt')['pixel_values'].half().cuda()
```
Comment (haotian-liu) [inline llava/serve/cli.py:55]: Why is this line removed?

**Review by haotian-liu — CHANGES_REQUESTED** [review]
Thanks. Can you check and see if this line is removed accidentally?

**Comment by eggry** [comment]
> Thanks. Can you check and see if this line is removed accidentally?

Reverted. Thanks for pointing out.

**Review by haotian-liu — APPROVED** [review]
LGTM
